jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/BuildImageTask.java

```diff
@@ -71,11 +95,93 @@ public boolean getReproducible() {
     return format;
   }
 
+  @Input
+  public boolean getUseOnlyProjectCache() {
+    return useOnlyProjectCache;
+  }
+
   @TaskAction
-  public void buildImage() {
-    // TODO: Implement.
+  public void buildImage() throws InvalidImageReferenceException, IOException {
+    // Asserts required @Input parameters are not null.
```

Comment:
I don't think @Input parameters can be null unless they're optional.

Reply:
Yea, this is just so that NullAway does not barf (compile-time null-checking cannot infer guarantee of `@Input` parameters not being null).